### PR TITLE
[Ikanji David]-Sage-Land auction tutorial sage

### PR DESCRIPTION
**Pull request by Ikanji201**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

 
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/03-20-2023-decentralized-land-auction-smart-contract/Decentralized-Land-Auction-Smart-Contract.md [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+
+After successfully creating the new file, the following step would be to specify some statements in our smart contract.
+
+```solidity
+ // SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0 <0.9.0;
+```
+ 
+ In the provided code, we use the statement `SPDX-License-Identifier`: MIT to indicate that the code is licensed under the MIT License. This is achieved through the use of the SPDX (Software Package Data Exchange) identifier, which is a standard method of identifying open-source licenses.
```
Comment (4undRaiser) [inline blog/03-20-2023-decentralized-land-auction-smart-contract/Decentralized-Land-Auction-Smart-Contract.md:193]: the beginning of each paragraph should start from the first line

File: blog/03-20-2023-decentralized-land-auction-smart-contract/Decentralized-Land-Auction-Smart-Contract.md [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+
+- `transfer`: transfers tokens from the sender's account to another account.
+
+- `approve`: approves a specific account to withdraw tokens from the sender's account.
+
+- `transferFrom`: allows the approved account to transfer tokens from the sender's account.
+
+- `totalSupply`: returns the total amount of tokens in existence.
+
+`balanceOf`: returns the balance of tokens in a specific account.
```
Comment (4undRaiser) [inline blog/03-20-2023-decentralized-land-auction-smart-contract/Decentralized-Land-Auction-Smart-Contract.md:221]: inconsistent list tags here. fix it

**Comment by 4undRaiser** [comment]
fix these and let me know

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Pull request merged**